tests/functional_tests/t0_main/monai/test_monai.yea

```diff
@@ -0,0 +1,31 @@
+id: 0.monai.segmentation
+plugin:
+    - wandb
+tag:
+  platforms:
+    - linux
+command:
+    program: test_monai.py
+depend:
+    requirements:
+        - "monai>=1.3.0"
+        - "pytorch-ignite>=0.4.11"
+        - "nibabel>=5.2.0"
+assert:
+    - :wandb:runs_len: 1
+    - :wandb:runs[0][config][epochs]: 2
+    - :wandb:runs[0][config][train_batch_size]: 2
+    - :wandb:runs[0][config][val_batch_size]: 5
+    - :op:>=:
+        - :wandb:runs[0][summary][epoch/Mean_Dice]
+        - 0
+    - :op:==:
+        - :wandb:runs[0][summary][epoch/epoch]
+        - 2
+    - :op:>=:
+        - :wandb:runs[0][summary][iteration/Loss]
+        - 0
+    - :op:==:
+        - :wandb:runs[0][summary][iteration/iteration]
+        - 20
+    - :wandb:runs[0][exitcode]: 0
```

Comment:
Curiously, it looks like it fails on ci, although yea doesn't seem to catch it:
https://app.circleci.com/pipelines/github/wandb/wandb/35609/workflows/51972e1e-d502-4eda-893d-e05d98e6dfb6/jobs/1201297?invite=true#step-105-13848_46